src/astro/databases/snowflake.py

```diff
@@ -506,13 +506,23 @@ def schema_exists(self, schema: str) -> bool:
 
         :param schema: DB Schema - a namespace that contains named objects like (tables, functions, etc)
         """
-        created_schemas = [
-            x["SCHEMA_NAME"]
-            for x in self.hook.run(
+
+        # Below code is added due to breaking change in apache-airflow-providers-snowflake==3.2.0,
+        # we need to pass handler param to get the rows. But in version apache-airflow-providers-snowflake==3.1.0
+        # if we pass the handler provider raises an exception AttributeError 'sfid'.
```

Comment:
I think this should be rather in the PR or commit message. But I like that you commented it in general :)